commit 5393ddf14e7071ceeefde74a95bbc6ac9203c579
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:50:20 2010 -0700

    Staging: brcm80211: typedefs.h: remove some unused #if logic
    
    We never care about __STRICT_ANSI__ from within the kernel, so remove
    this logic.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -32,13 +32,6 @@ typedef unsigned long long int uintptr;
 #define TYPEDEF_USHORT
 #define TYPEDEF_ULONG
 #include <linux/version.h>
-
-/* Do not support the (u)int64 types with strict ansi for GNU C */
-#if defined(__GNUC__) && defined(__STRICT_ANSI__)
-#define TYPEDEF_INT64
-#define TYPEDEF_UINT64
-#endif
-
 #include <linux/types.h>
 
 /*
